tests/components/islamic_prayer_times/test_init.py

```diff
@@ -57,7 +55,7 @@ async def test_setup_failed(hass: HomeAssistant) -> None:
 
     # test request error raising ConfigEntryNotReady
     with patch(
-        "prayer_times_calculator.PrayerTimesCalculator.fetch_prayer_times",
+        "prayer_times_calculator_offline.PrayerTimesCalculator.fetch_prayer_times",
```

Comment:
I assume this InvalidResponseError cannot happen anymore

Reply:
Yes, thank you for pointing this out. The exception is still included in `prayer_times_calculator_offline` for backwards compatibility with `prayer_times_calculator` (perhaps there are non-HA users out there?), but it will never be raised.

I neglected to clean up all instances of it in this PR - addressed in f9fe9e3.